src/components/charts/sizeDistancePlotter/LegendMultiple.jsx

```diff
@@ -37,7 +38,7 @@ class LegendMultiple extends React.PureComponent {
                   }
                 ></span>
                 <span>{`${
-                  isDefaultSet ? 'Rubin Observatory Data' : 'Your Data'
+                  isDefaultSet ? 'Rubin Observatory Data' : datum.group
```

Comment:
after getting rid of `isDefaultSet` (see other comments) this probably becomes `datum.group || 'Rubin Observatory Data'`